scripts/tutorial_3.py

```diff
@@ -20,8 +20,20 @@ def main(args=None):
     spin_thread.start()
     interface.start_competition()
 
-    #TODO
+    # Turns on a debug topic to visualize bounding boxes and slots
+    # /ariac/sensors/display_bounding_boxes
+    interface.display_bounding_boxes = True
+    
+    bin_number = 2
 
+    while rclpy.ok():
+        try:
+            # publishes parts in the bin on /ariac/sensors/slot_occupancy
+            interface.get_bin_parts(bin_number)
```

Comment:
get_bin_parts() should return the dictionary and the logging should happen at this level.